docs/guides/how-to-reset-state.md

```diff
@@ -44,8 +44,8 @@ const useSlice = create<State & Actions>()((set, get) => ({
 Resetting multiple stores at once
 
 ```ts
-import { create as _create } from 'zustand'
-import type { StateCreator } from 'zustand'
+import type * as zustand from 'zustand'
```

Comment:
Personally, I'm not a big fan of this. Can we keep the previous style?

Reply:
Sure